src/System.Memory/ref/System.Memory.cs

```diff
@@ -112,6 +112,24 @@ public static partial class MemoryExtensions
 }
 namespace System.Buffers
 {
+    public sealed partial class ArrayBufferWriter : System.Buffers.IBufferWriter<byte>, System.IDisposable
+    {
+        public ArrayBufferWriter() { }
+        public ArrayBufferWriter(int initialCapacity) { }
+        public int BytesAvailable { get { throw null; } }
+        public int BytesWritten { get { throw null; } }
+        public int Capacity { get { throw null; } }
+        public System.ReadOnlyMemory<byte> OutputAsMemory { get { throw null; } }
+        public System.ReadOnlySpan<byte> OutputAsSpan { get { throw null; } }
+        public long TotalBytesWritten { get { throw null; } }
+        public void CopyToAndReset(System.IO.Stream stream) { }
+        public System.Threading.Tasks.Task CopyToAndResetAsync(System.IO.Stream stream, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
@stephentoub, should this return a `ValueTask` instead?

Reply:
As I noted on my other comment, I don't think we should have these copy methods at all.